client/verta/verta/_internal_utils/_utils.py

```diff
@@ -283,16 +283,23 @@ def find(self, where):
         --------
         .. code-block:: python
 
+            runs.find("hyperparameters.hidden_size == 256",
+                       "metrics.accuracy >= .8")
+            # <ExperimentRuns containing 3 runs>
+            # alternatively:
             runs.find(["hyperparameters.hidden_size == 256",
                        "metrics.accuracy >= .8"])
             # <ExperimentRuns containing 3 runs>
 
         """
-        new_list = copy.deepcopy(self)
+        if len(args) == 1 and isinstance(args[0], list):
+            # to keep backward compatibility, in case user pass in a list
+            return self.find(*args[0])
+        elif not all([isinstance(predicate, six.string_types) for predicate in args]):
```

Comment:
Listification isn't necessary for `all()`
```suggestion
        elif not all(isinstance(predicate, six.string_types) for predicate in args):
```

Reply:
Fixed.